aiodocker/services.py

```diff
@@ -87,11 +87,53 @@ def __init__(self, docker):
         response = await self.docker._query_json(
             "services/create",
             method="POST",
-            headers={"Content-type": "application/json", },
+            headers={"Content-type": "application/json"},
             data=data,
         )
         return response
 
+    async def update(self,
+                     service_id: str,
+                     *,
+                     image: str,
+                     rollback: bool=False
```

Comment:
Probably good to have rollback test case also

Reply:
Added.